types/src/shared/rate_limiter.ts

```diff
@@ -80,6 +92,28 @@ export async function rateLimiter({
   }
 }
 
+export async function expireRateLimiterKey({
+  key,
+  redisUri,
+}: RateLimiterOptionsBase): Promise<Result<boolean, Error>> {
+  let redis: undefined | Awaited<ReturnType<typeof redisClient>> = undefined;
+
+  try {
+    redis = await getRedisClient(redisUri);
+    const redisKey = makeRateLimiterKey(key);
```

Comment:
Shouldn't this be done in `makeMessageRateLimitKeyForWorkspace`?

Reply:
It adds the `rate_limiter:` prefix.

Reply:
Makes kinda sense to do it within `makeMessageRateLimitKeyForWorkspace` then, no? 😸